server/routes/api.js

```diff
@@ -484,11 +486,13 @@ router.post('/messages', function(req, res, next) {
                         res.status(200);
                         res.send('Ignoring duplicate');
                     } else {
-                        db.get("SELECT id, ignore, push, pushpri, mailenable, mailto FROM capcodes WHERE ? LIKE address ORDER BY REPLACE(address, '_', '%') DESC LIMIT 1", address, function(err,row) {
+                        db.get("SELECT id, ignore, push, pushpri, pushgroup, pushsound, mailenable, mailto FROM capcodes WHERE ? LIKE address ORDER BY REPLACE(address, '_', '%') DESC LIMIT 1", address, function(err,row) {
                             var insert;
                             var alias_id = null;
                             var pushonoff = null;
                             var pushpri = null;
+                            var pushgroup = null;
```

Comment:
There's a bit of a weird mix of casing here and 508/509. I see it was present in the previous commits for the pushover stuff, so probably all good. I don't have pushover so can't test properly - as long as it's all good for you then I'll merge, just wanted to highlight in case it was unintentional :)

Reply:
I'm shocking at this - I'm used to working in powershell where case doesn't matter ;)

From memory i matched it to the casing of the rest of the pushover stuff as you said.